### PR TITLE
initial version of confirm dialog

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

**Comment by srambach** [comment]
Should the first two stories for this display the modal?
By default, as a confirmation modal, I think this should use a primary button, and a variant added to make it a danger button.
I think we should also make the modal small by default, and allow a variant for the other sizes if needed. (unless you have mockups saying otherwise?)

File: src/components/confirm-dialog/ConfirmDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          }}
+        >
+          {t(continueButtonLabel || "common:continue")}
+        </Button>,
+        <Button
+          id="modal-cancel"
+          key="cancel"
+          variant={ButtonVariant.secondary}
+          onClick={() => {
+            if (onCancel) onCancel();
```
Comment (ssilvert) [inline src/components/confirm-dialog/ConfirmDialog.tsx:76]: If there is no cancel handler, we shouldn't snow the cancel button.

**Review by ssilvert — CHANGES_REQUESTED** [review]

**Comment by ssilvert** [comment]
> Should the first two stories for this display the modal?
> By default, as a confirmation modal, I think this should use a primary button, and a variant added to make it a danger button.
> I think we should also make the modal small by default, and allow a variant for the other sizes if needed. (unless you have mockups saying otherwise?)

Agree with @srambach.  Otherwise, LGTM.

**Review by dlabaj — APPROVED** [review]
LGTM other then sarah's comment

**Comment by edewit** [comment]
@srambach ahh yes it should show the dialog, fixed that now

**Pull request merged**